### PR TITLE
Add New Featured Artist: Lifetheory newspost

### PR DESCRIPTION
**Pull request by Ephemeralis**
🎵

### CODE REVIEW

File: news/2020-12-09-new-featured-artist-lifetheory.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+Check out another entry in the trilogy with [this classic 2013 set](https://osu.ppy.sh/beatmapsets/113192) mapped by [Zare](https://osu.ppy.sh/users/1437117)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/122/Daisy/Lifetheory%20-%20Angel.mp3" type="audio/mpeg">
+</audio>
+
+### Fairy
+
+Check out the final entry in the trilogy with [this classic 2014 set](https://osu.ppy.sh/beatmapsets/113192) mapped by [Zare](https://osu.ppy.sh/users/1437117)!
```
Comment (Nivalyx) [inline news/2020-12-09-new-featured-artist-lifetheory.md:43]: the link is wrong here, it should link to https://osu.ppy.sh/beatmapsets/158820 instead

File: news/2020-12-09-new-featured-artist-lifetheory.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/artists/122/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+Still need more convincing? We've got you covered - check out the trilogy of maps that started it all:
+
+### Daisy
+
+Check out [this classic 2013 set](https://osu.ppy.sh/beatmapsets/118494) mapped by [Zare](https://osu.ppy.sh/users/1437117)!
```
Comment (cl8n) [inline news/2020-12-09-new-featured-artist-lifetheory.md:27]: I think it'd read nicer to include the "mapped by zare" and "classic set" parts in L23, and leave these sections empty besides the audio. cuts out the repetition of "classic set" and "trilogy"

if applied I guess that means the headers would have to be the map links instead

File: news/2020-12-09-new-featured-artist-lifetheory.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    <source src="https://assets.ppy.sh/artists/122/Daisy/Lifetheory%20-%20Fairy.mp3" type="audio/mpeg">
+</audio>
+
+---
+
+Like what you see/hear? Check out [**Lifetheory**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/122) for a whole load more, all pre-packaged and ready for some life via the application of mapping theory.
+
+That's all for today, but we'll be back again (as usual) on Saturday with another awesome artist, so make sure to stay in the know! The [@osugame Twitter](https://twitter.com/osugame) is an excellent place to do that, if you're not following it already.
+
+—Ephemeral & pishifat
```
Comment (cl8n) [inline news/2020-12-09-new-featured-artist-lifetheory.md:55]:
```suggestion
—Ephemeral and pishifat
```
i know it looks a little worse here, but the more prominent top of news page will end up reading "by Ephemeral and pishifat". + consistency with other posts

File: news/2020-12-09-new-featured-artist-lifetheory.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+In case you weren't with us yet when the **Lifetheory** craze was at its peak, check it all out in a little medley style preview below, courtesy of the most pishi of fats:
+
+<div align="center">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/artists/122/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+Still need more convincing? We've got you covered - check out the trilogy of maps that started it all:
```
Comment (cl8n) [inline news/2020-12-09-new-featured-artist-lifetheory.md:23]:
```suggestion
Still need more convincing? We've got you covered—check out the trilogy of maps that started it all:
```
your favorite review

**Review by peppy — APPROVED** [review]

**Pull request merged**